### ISSUE TITLE
dataset distribute

### ISSUE THREAD

**Issue by winnieepooh** [jantic/DeOldify#478]
Hi,
How is the dataset distributed in the training process? Are the train set and the valid set used to train the generator and the test set used to train the critic?
Thanks.

**Comment by winnieepooh** [comment]
Which data set becomes a grayscale image when the create_training_images function is executed?

**Comment by winnieepooh** [comment]
Which dataset becomes  grayscale images when the create_training_images function is executed?

**Comment by jantic** [comment]
Training set is used for all the training.  I used ImageNet, but I'd suggest trying OpenImages these days (more diverse and larger dataset).
